src/py/flwr/client/supernode/app.py

```diff
@@ -44,11 +47,19 @@ def run_supernode() -> None:
 
     event(EventType.RUN_SUPERNODE_ENTER)
 
-    _ = _parse_args_run_supernode().parse_args()
+    args = _parse_args_run_supernode().parse_args()
 
-    log(
-        DEBUG,
-        "Flower SuperNode starting...",
+    root_certificates = _get_certificates(args)
+    load_fn = _get_load_client_app_fn(args, multi_app=True)
+
+    _start_client_internal(
```

Comment:
If we are introducing `flower-supernode` cli soon; should we throw a deprecation warning if `_start_client_internal` gets trigger from a `flower-client-app` command? (i.e. add the warning in the `run_client_app()` function a few lines below. Thoughts @danieljanes @panh99 ?

Reply:
@jafermarq by that, do you mean: should we deprecate `flower-client-app`?

Reply:
yes, that's a better way to put it.

Reply:
Yes. How about we do this with the Flower 1.11 release? This would give us one release (Flower 1.10) with both `flower-supernode` and `flower-client-app` not deprecated. We can then change all the docs to `flower-supernode` and then release Flower 1.11 with a deprecated `flower-client-app`.

Reply:
Sound good to me!